src/components/SetupOpportunityPage/SetupOpportunityPage.module.scss

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
These module.scss files aren't needed now. They shouldn't be in the hygen templates anymore